backend/application/syson-application/src/test/java/org/eclipse/syson/application/controllers/diagrams/general/view/GVEdgeItemUsageTests.java

```diff
@@ -0,0 +1,269 @@
+public class GVEdgeItemUsageTests extends AbstractIntegrationTests {
+
+    private static final String ITEM1 = "item1";
+
+    private final IDescriptionNameGenerator descriptionNameGenerator = new GVDescriptionNameGenerator();
+
+    @Autowired
+    private IGivenInitialServerState givenInitialServerState;
+
+    @Autowired
+    private IGivenDiagramReference givenDiagram;
+
+    @Autowired
+    private IGivenDiagramDescription givenDiagramDescription;
+
+    @Autowired
+    private IGivenDiagramSubscription givenDiagramSubscription;
+
+    @Autowired
+    private IDiagramIdProvider diagramIdProvider;
+
+    @Autowired
+    private NodeCreationTester nodeCreationTester;
+
+    @Autowired
+    private DiagramComparator diagramComparator;
+
+    @Autowired
+    private IObjectSearchService objectSearchService;
+
+    @Autowired
+    private SemanticRunnableFactory semanticRunnableFactory;
+
+    private DiagramDescriptionIdProvider diagramDescriptionIdProvider;
+
+    private DiagramCheckerService diagramCheckerService;
+
+    private StepVerifier.Step<DiagramRefreshedEventPayload> verifier;
+
+    private SemanticCheckerService semanticCheckerService;
+
+    private AtomicReference<Diagram> diagram;
+
+    @BeforeEach
+    public void setUp() {
+        this.givenInitialServerState.initialize();
+        var diagramEventInput = new DiagramEventInput(UUID.randomUUID(),
+                GeneralViewWithTopNodesTestProjectData.EDITING_CONTEXT_ID,
+                GeneralViewWithTopNodesTestProjectData.GraphicalIds.DIAGRAM_ID);
+        var flux = this.givenDiagramSubscription.subscribe(diagramEventInput);
+        this.verifier = StepVerifier.create(flux);
+        this.diagram = this.givenDiagram.getDiagram(this.verifier);
+        var diagramDescription = this.givenDiagramDescription.getDiagramDescription(GeneralViewWithTopNodesTestProjectData.EDITING_CONTEXT_ID,
+                SysONRepresentationDescripionIdentifiers.GENERAL_VIEW_DIAGRAM_DESCRIPTION_ID);
+        this.diagramDescriptionIdProvider = new DiagramDescriptionIdProvider(diagramDescription, this.diagramIdProvider);
+        this.diagramCheckerService = new DiagramCheckerService(this.diagramComparator, this.descriptionNameGenerator);
+        this.semanticCheckerService = new SemanticCheckerService(this.semanticRunnableFactory, this.objectSearchService, GeneralViewWithTopNodesTestProjectData.EDITING_CONTEXT_ID,
+                GeneralViewWithTopNodesTestProjectData.SemanticIds.PACKAGE_1_ID);
+    }
+
+    @AfterEach
+    public void tearDown() {
+        if (this.verifier != null) {
+            this.verifier.thenCancel()
+                    .verify(Duration.ofSeconds(10));
+        }
+    }
+
+    @DisplayName("Given a SysML Project with an Action Usage, when creating ItemUsage with direction in, then they should be displayed as BorderedNode and ItemList in the nestedItems comparement")
```

Comment:
BorderedNode => bordered node
ItemList => list item
comparement => compartment

Please fix all other @DisplayName in this file

Reply:
Done.